### ISSUE TITLE
crach

### ISSUE THREAD

**Issue by fourmis-84** [Chicken-Bones/NotEnoughItems#416]
my game crashes when I launch it because of this mod.

I am in version 1.7.10

Thank you

**Comment by AlexMinecraft2012** [comment]
Did you miss CodeChickenCore.

**Comment by fourmis-84** [comment]
No, I think it's a problem of vercion between nei and CodeChickenCore.
Thank you
